### DISCUSSION TITLE
My account got hacked ! please help!

### DISCUSSION THREAD

**Discussion by mill3rtime1** [community/community#114775] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

I went to go access my account for the first time in a while and the email has been changed. I no longer have access and the links to my work do not work. Please help!

**Comment by queenofcorgis** [comment, marked as answer]
Hi @mill3rtime1 ! The best place to go for answers to account-specific questions like this one is opening a ticket on our [Support page](https://support.github.com/contact).

Please note: The community is unable to assist with nor escalate your ticket, and it will be answered in the order it was received based on our team's resources and the amount of tickets ahead of yours. We appreciate your understanding!

As no other Community Discussions team members will be able to provide additional help with account related questions I am going to close this discussion.

Thank you!
